org-node/src/client.rs

```diff
@@ -282,69 +521,30 @@ impl Client {
                 let mut peers = api.providers(urn.clone(), timeout);
 
                 while let Some(peer) = peers.next().await {
-                    match Client::track_project(api, &urn, &peer).await {
-                        Err(err) => {
-                            tracing::error!(target: "org-node", "Error tracking {}: {}", urn, err);
+                    // Send notification to track peers process;
+                    let response = match Client::track_peer_handler(
```

Comment:
Maybe move this out to make the code a little easier to read:
```rust
let result = Client::track_peer_handler(...).await;
let response = match result { ...
```

Reply:
5dfbfd6 fixes.